plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/query/TestQueryOptionsParsing.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.pinot.query;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestQueryOptionsParsing
```

Comment:
Remove `public`, add `final` and rename to `TestPinotQueryBuilder`. 

> Test classes should be defined as package-private and final.
https://trino.io/docs/current/develop/tests.html#conventions-and-recommendations